.github/workflows/main.yml

```diff
@@ -0,0 +1,99 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["2.7", "3.5", "3.6", "3.7"]
+        ansible:
+          - stable-2.7
```

Comment:
This is going to be a collection to begin with, right?
Then you should not test ansible 2.7
However 2.10 would be interesting. ;)